ext/autocomplete/style.css

```diff
@@ -0,0 +1,8 @@
+#Navigationleft .blockbody { overflow: visible; }
+
+.tagit { background: white !important; border: 1px solid grey !important; cursor: text; }
+.tagit-choice { cursor: initial; }
+input[name=search] ~ input[type=submit] { display: block !important; }
```

Comment:
Why `display:block`? `inline-block` seems nicer on the things I'm looking at (eg, on the home page, block causes the input box and the submit button to be on different lines, with the submit button floating off against one edge of the screen)